### DISCUSSION TITLE
Is it not possible to run the output index.html as standalone html without deploying it?

### DISCUSSION THREAD

**Discussion by AsuraKev** [web-infra-dev/modern.js#4663] [Q&A]
Hi there, I have a question regarding the output index.html

We have a use case where we can directly copy and paste the index.html into a website's html widget and the site will just display.
I am wondering if the same can be done with the index.html generated by Modern.js?

I am seeing 404 :( 

![image](https://github.com/web-infra-dev/modern.js/assets/52765572/6fa32980-1ea7-43ef-817d-ddc1dbff8702)

thanks

**Comment by chenjiahan** [comment]
You can inline the script and style files into the HTML file. This way you get an HTML file that can run independently.

- [output.enableInlineScripts](https://modernjs.dev/en/configure/app/output/enable-inline-scripts.html)
- [output.enableInlineStyles](https://modernjs.dev/en/configure/app/output/enable-inline-styles.html)

But this will still have many limitations, for example React Router may not work properly.

**Reply by AsuraKev** [reply]
cool thanks!
